www/src/components/page-with-plugin-searchbar.js

```diff
@@ -20,7 +20,7 @@ const PageWithPluginSearchBar = ({ isPluginsIndex, location, children }) => (
       css={{
         ...styles.content,
         // mobile: hide README on gatsbyjs.org/plugins index page
-        display: `${isPluginsIndex && `none`}`,
+        display: isPluginsIndex ? `none` : false,
```

Comment:
_Could_ also solve like so, both work though 🤷‍♂️ 

```suggestion
        ...(isPluginsIndex && { display: 'none' })
```

Reply:
💯